crates/model/src/solver_competition.rs

```diff
@@ -24,7 +24,8 @@ pub struct Request {
     pub competition: SolverCompetitionDB,
     pub executions: Vec<(OrderUid, Execution)>,
     pub scores: Scores,
-    pub participants: Vec<H160>, // solver addresses
+    pub participants: Vec<H160>,      // solver addresses
```

Comment:
This trivially ensures that we'll always have unique participants in the DB.
```suggestion
    pub participants: HashSet<H160>,      // solver addresses
```

Reply:
Fixed.